### ISSUE TITLE
Clean JSON output possible?

### ISSUE THREAD

**Issue by lauritzsh** [wackou/guessit#237]
Right now when I use the `--advanced` flag for JSON output, it also includes unnecessary information (IMO) that makes it hard to parse the JSON. The Breaking Bad example:

```json
For: Breaking.Bad.S05E08.720p.MP4.BDRip.[KoTuWa].mkv
GuessIt found: {
    "mimetype": {
        "confidence": 1.0,
        "value": "video/x-matroska"
    },
    "episodeNumber": {
        "raw": "S05E08",
        "confidence": 0.6,
        "value": 8
    },
    "container": {
        "raw": "MP4",
        "confidence": 0.937,
        "value": "mkv"
    },
    "format": {
        "raw": "BDRip",
        "confidence": 0.6,
        "value": "BluRay"
    },
    "series": {
        "raw": "Breaking.Bad",
        "confidence": 0.42,
        "value": "Breaking Bad"
    },
    "releaseGroup": {
        "raw": "[KoTuWa]",
        "confidence": 0.17189999999999994,
        "value": "KoTuWa"
    },
    "screenSize": {
        "raw": "720p",
        "confidence": 0.6,
        "value": "720p"
    },
    "season": {
        "raw": "S05E08",
        "confidence": 0.6,
        "value": 5
    },
    "type": {
        "confidence": 1.0,
        "value": "episode"
    }
}
```

Could it be possible to have a `--json` flag that _only_ prints out that JSON so it's easy to parse? I could give it a shot, but I'm not familiar with Python to be honest.

**Comment by Toilal** [comment]
It has been implemented in guessit 2 which is currently released in alpha.
I'll backport the feature to guessit 1.
Le 8 nov. 2015 7:08 PM, "Lauritz Hilsøe" <notifications@github.com> a
écrit :

**Comment by lauritzsh** [comment]
Awesome, thanks for the quick reply! I'll give guessit 2 a chance, would you say it's stable enough for use (personal project) or should I wait for a stable version?

**Comment by Toilal** [comment]
You can use it now ! It's really stable from my point of view 😉. Use
`--pre` flag on pip install.
Le 8 nov. 2015 7:17 PM, "Lauritz Hilsøe" <notifications@github.com> a
écrit :

**Comment by lauritzsh** [comment]
Thanks, just gave it a try and works great! I'll close it since it is already implemented. :+1:
